### PR TITLE
Fix broken status on Travis CI

### PR DESCRIPTION
**Pull request by hsbt**
In Dec 2017, 1-16-stable branch of Travis was broken caused outside condition. I try to fix it.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -101,13 +101,19 @@
...
   original_env = ENV.to_hash.delete_if {|k, _v| k.start_with?(Bundler::EnvironmentPreserver::BUNDLER_PREFIX) }
+  original_default_specs = Dir[File.join(Gem.default_dir, "specifications", "default", "bundler*")]
 
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
 
   config.before :all do
     build_repo1
+    original_default_specs.each {|s| FileUtils.mv(s, s + ".org") }
```
Comment (segiddins) [inline spec/spec_helper.rb]: why do we need this?

Comment (hsbt) [reply spec/spec_helper.rb]: some examples that use fake bundle commands are failing status when there is bundler as default gems.

see https://travis-ci.org/bundler/bundler/jobs/313289214

**Comment by hsbt** [comment]
I'm going to `git squash` before merging working commits. These changes is that bundler installation as default gems move and restore during rspec invoking.

@segiddins Do you have another comment? I hope to finish to prepare bundler-1.16.1 releasing until Dec 12.

**Comment by hsbt** [comment]
@bundlerbot r+

**Comment by hsbt** [comment]
@bundlerbot retry

**Pull request merged**